spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/nosql.adoc

```diff
@@ -151,6 +151,13 @@ TIP: For complete details of Spring Data MongoDB, including its rich object mapp
 Spring Boot offers auto-configuration for https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo[Embedded Mongo].
 To use it in your Spring Boot application, add a dependency on `de.flapdoodle.embed:de.flapdoodle.embed.mongo`.
 
+The version of the MongoDB server **must** be configured by setting the configprop:spring.data.mongodb.embedded.version property.
+
+NOTE: The default download configuration allows access to most of the versions listed in https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/blob/de.flapdoodle.embed.mongo-3.0.0/src/main/java/de/flapdoodle/embed/mongo/distribution/Version.java[Version.java]
+as well as some others. Inaccessible versions result in error when attempting to download the server. The download configuration may be adjusted to handle the desired version.
```

Comment:
I am not sure what the recommended way is to point to the `Version` source file from the docs. 

I pointed to the 3.0.0 tag since that is what the main Spring Boot branch currently points to for flapdoodle. This makes the info in that particular file accurate but also makes a location that needs to be updated when the flapdoodle version changes. However, If we use link to the flapdoodle main branch then the info may not be accurate as the Version file may change over time.

Reply:
Just in case you're interested, @bono007, I used an attribute for this with its value set to the version of the embedded Mongo dependency: https://github.com/spring-projects/spring-boot/commit/4d76660f1735f34cf095b44c983ce99fae269dd5#diff-ad4cb147eace80b1ffbafcd284f6c2c8479a07ecc150d71f49eb34ce6d37293dR223.

Reply:
> Just in case you're interested, @bono007, I used an attribute for this with its value set to the version of the embedded Mongo dependency: [4d76660#diff-ad4cb147eace80b1ffbafcd284f6c2c8479a07ecc150d71f49eb34ce6d37293dR223](https://github.com/spring-projects/spring-boot/commit/4d76660f1735f34cf095b44c983ce99fae269dd5#diff-ad4cb147eace80b1ffbafcd284f6c2c8479a07ecc150d71f49eb34ce6d37293dR223).

TIL! Thx for pointing that out as I will use this in the future.